content/klc/G1.8.adoc

```diff
@@ -1,7 +1,8 @@
 +++
-brief = "Components can only contain features supported in latest stable release"
+brief = "Contributions to the official lib must be made using the current stable version"
 +++
 
-KiCad library components (symbols, footprints) may only contain features that are supported by the latest __stable__ KiCad software release.
+Contributions of library assets (symbols and footprints) must be created in the latest **stable** version of KiCad.
```

Comment:
Worksheets, templates, etc. are also in here. Maybe just easier to remove the entire parentheses stuff?